cdap-docs/developer-manual/source/building-blocks/schedules.rst

```diff
@@ -49,6 +51,22 @@ the same workflow is executing at that time. Note that:
 - This schedule does not configure a timeout, such that the default timeout of one day
   is used.
 
+If more than one conditions are required to construct the *Trigger* for the schedule,
+to an application extended from ``AbstractApplication``, get a *TriggerFactory* from
+`schedule
+<../../reference-manual/javadocs/co/cask/cdap/api/app/AbstractApplication.html#getTriggerFactory()>`__
+to create a specific *Trigger* to be used as follows::
+
+  schedule(
+    buildSchedule("Workflow1AndWorkflow2CompletedSchedule", ProgramType.WORKFLOW, "TriggeredWorkflow")
+      .triggerOn(getTriggerFactory().and(getTriggerFactory().onProgramStatus(ProgramType.WORKFLOW, "Workflow1",
+                                                                             ProgramStatus.COMPLETED),
+                                         getTriggerFactory().onProgramStatus(ProgramType.WORKFLOW, "Workflow2",
+                                                                             ProgramStatus.COMPLETED)));
+
+This schedule uses an and trigger that can only be satisfied when both Workflow1
```

Comment:
I feel like the "and" should be wrapped in with a `, or quotes or something special, otherwise it can be hard (or confusing) to read the sentence.

Reply:
or uppercase `AND`?